[Bug]: LUE fails to build against HPX development version/version 2.x

### Describe the bug

configure fails with 
```CMake Error at environment/cmake/LueConfiguration.cmake:537 (find_package):
  Could not find a configuration file for package "HPX" that is compatible
  with requested version range "1.11...<1.12".
```


### Expected behaviour

_No response_

### Steps to reproduce

_No response_

### Additional information

_No response_

--- a/environment/cmake/LueConfiguration.cmake
+++ b/environment/cmake/LueConfiguration.cmake
@@ -534,7 +534,7 @@ endif()
 
 
 if(LUE_HPX_REQUIRED)
-    find_package(HPX 1.11...<1.12 REQUIRED)
+    find_package(HPX 1.11 REQUIRED)
 
     if(HPX_FOUND)
         message(STATUS "Using HPX ${HPX_VERSION} found in ${HPX_PREFIX}")

